ideascube/utils.py

```diff
@@ -1,8 +1,27 @@
+from io import TextIOWrapper
 import sys
 
 from django.conf import locale
 
 
+class AutomagicTextIOWrapper(TextIOWrapper):
```

Comment:
This is why I suggest to just put it on the class itself: curious people will end there if they want to understand why we use it :)
About the name: not sure we need to keep the reference to TextIoWrapper in it. I'd go for something like IterDecode, because that's what it does for us. Thoughts?